Webex Connect Flows/v3.4/Template/Media Specific Workflows/README.md

```diff
@@ -0,0 +1,102 @@
+# Media Specific Workflows
+
+## Workflows Overview
+The following workflow templates are included :
+
+**Media Specific Workflows** : These flows need to be imported once per asset of the specific channel.
+* **Facebook Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated Facebook page.
+* **Live Chat Inbound Flow** - This workflow will be triggered for every inbound customer message over the configured customer chat widget, and the widget has a live chat form configured.
+* **Live Chat Inbound Flow Without Form** - This workflow will be triggered for every inbound customer message over the configured customer chat widget and the widget doesn't have a live chat form configured.
+* **SMS Inbound Flow** - This workflow will be triggered for every inbound customer sms sent over the integrated SMS phone number.
+* **Email Inbound Flow** - This workflow will be triggered for every inbound customer email to the integrated email account.
+* **Whatsapp Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated whatsapp business number.
+* **Live Chat Close Flow** - This workflow will be triggered when a livechat customer has ended the chat.
+* **GBM Inbound Flow** - This workflow will be triggered for every inbound customer message over the configured Google Business Messages Agent.
+* **GBM Inbound Flow With Suggestion** - This workflow will be triggered for every inbound customer message over the configured Google Business Messages Agent and will offer the customer a few suggestions.
+
+Note : 
+- Queue selection from queue task node is mandatory for any channel flows before making the flow live.
+- Flow variables must be updated in the setting section with the correct asset details before making the flow live.
```

Comment:
>Flow variables must be updated in the setting section with the correct asset details before making the flow live.

Can we please check this? Should it be like
Flow variables must be updated in the setting section with the correct details before making the flow live.